libr/arch/p/arm/armass64.c

```diff
@@ -84,9 +88,12 @@ typedef struct Opcode_t {
 	Operand operands[MAX_OPERANDS];
 } ArmOp;
 
-#define check_cond(cond) if (!(cond)) { return data; }
+#define check_cond(cond) \
```

Comment:
one line reads better with grep, also multiline defines should be avoided if necessary

Reply:
I have made this one-line using the GitHub web GUI, which hopefully works for now